snuba/settings/__init__.py

```diff
@@ -218,7 +218,8 @@ class RedisClusters(TypedDict):
 BULK_CLICKHOUSE_BUFFER = 10000
 BULK_BINARY_LOAD_CHUNK = 2**22  # 4 MB
 
-USE_EAP_ITEMS_TABLE = os.environ.get("USE_EAP_ITEMS_TABLE", False)
+USE_EAP_ITEMS_TABLE = os.environ.get("USE_EAP_ITEMS_TABLE", True)
+USE_EAP_ITEMS_TABLE_START_TIMESTAMP_SECONDS = 1741762800
```

Comment:
nit: good to leave a comment for the reader telling them what this date actually is so they don't have to convert it themselves. Also good to clarify when this setting can be removed